apps/src/lab2/views/components/Instructions/ResourcePanel/styles.module.scss

```diff
@@ -10,7 +10,7 @@
   justify-content: space-between;
 
   .tabs {
-    margin-top: 2rem;
+    margin-top: calc(2.5rem - 1px);
```

Comment:
What's the -1px for? border?

Reply:
Yeah without it the tab outline doesn't line up with the instruction panel border:
<img width="779" height="434" alt="image" src="https://github.com/user-attachments/assets/1ec82837-d7b8-4093-857c-55b75f5dc6e0" />

Reply:
(certainly possible there's a better way to do this 😅  -- I expect resource panel tab formatting is going to change with the new mocks that design has proposed)

Reply:
Yep seems fine for now!